spec/models/publication_spec.rb

```diff
@@ -161,9 +161,9 @@
 
     it 'should update attributions of existing contributions to the database' do
       expect(publication.contributions.size).to eq(0)
-      publication.contributions.build_or_update(author, status: 'new')
+      publication.contributions.build_or_update(author, status: 'unknown')
```

Comment:
What's happened here, any idea why this needs to change?

Reply:
It doesn't have to, but I think it reflects the intent of the test more clearly.  I.E., a different contribution w/ a different value.

Reply:
OK